app/ui/browser-modern/views/browser/tabbar/tab.jsx

```diff
@@ -103,7 +106,11 @@ class Tab extends Component {
   }
 
   render() {
-    return (
+    const {
+      connectDropTarget = _ => _,
+      connectDragSource = _ => _,
```

Comment:
Could definitely switch to default props.  Still would need the weirdness with returning the first arg though because of the way this wraps the return value in `render`